### PR TITLE
Update releases for new Tekton Pipeline Releases 0.63

### PR DESCRIPTION
**Pull request by chitrangpatel**
This is a documentation update to the new release.

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -41,11 +41,17 @@ Further documentation available:
...
+### v0.63
+- **Latest Release**: [v0.63.0][v0.63-0] (2024-08-30) ([docs][v0.63-0-docs], [examples][v0.63-0-examples])
+- **Initial Release**: [v0.63.0][v0.63-0] (2024-08-30)
+- **End of Life**: 2024-09-30
+- **Patch Releases**: [v0.63.0][v0.63-0]
+
+### v0.62 (LTS)
+- **Latest Release**: [v0.62.2][v0.62-2] (2024-08-23) ([docs][v0.62-2-docs], [examples][v0.62-2-examples])
+- **Initial Release**: [v0.62.0][v0.62-0] (2024-07-23)
 - **End of Life**: 2024-07-25
```
Comment (AlanGreene) [inline releases.md:53]: EOL for v0.62 should be 1 year after release as it's an LTS release, i.e. 2025-07-23

**Review by AlanGreene — CHANGES_REQUESTED** [review]

File: releases.md (hunk 1 above)
Comment (chitrangpatel) [reply releases.md]: Fixed!

**Review by vdemeester — APPROVED** [review]

**Review by afrittoli — COMMENTED** [review]
/lgtm

**Pull request merged**